.github/ISSUE_TEMPLATE/new_dataset.yaml

```diff
@@ -0,0 +1,15 @@
+name: 🗃New dataset
+description: Create a request for a new dataset to be added to MTEB
+title: "dataset: "
+labels: ["new dataset"]
+body:
+  - type: textarea
+    attributes:
+      label: dataset link on Hugging dataset
+      description: Please provide a link to the dataset on Hugging Face.
```

Comment:
I would reformulate to "Please provide a link to the dataset, ideally to Hugging Face."